docs/install/tarball.md

```diff
@@ -8,6 +8,44 @@ There are two tarballs available:
 
 * `percona-server-mongodb-{{release}}-x86_64.glibc2.35.tar.gz` is the tarball for Ubuntu 22.04.
 
+To check which `glibc` version your system is using, run the following command:
+
+```{.bash data-prompt="$"}
+$ ldd --version
+```
+
+??? example "Sample output"
+
+    ```{.text .no-copy}
+    ldd (GNU libc) 2.28
+    Copyright (C) 2018 Free Software Foundation, Inc.
+    This is free software; see the source for copying conditions.  There is NO
+    warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+    Written by Roland McGrath and Ulrich Drepper.
+    ```
+
+### What tarballs to choose
```

Comment:
Should probably be headline 2? We are going from 1 to 3, and then go back to 2 for "Precondition".  Check also "Tarball types".

Reply:
Fixed